mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -317,7 +317,7 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 		ARG 5 z
 		ARG 7 particleEffects
 	METHOD method_6083 getLastAttackTime ()I
-	METHOD method_6084 hasStackEquipped (Lnet/minecraft/class_1304;)Z
+	METHOD method_6084 hasStack (Lnet/minecraft/class_1304;)Z
```

Comment:
`hasStack` would imply "including inventory" by default. Sure, you will quickly understand that it does not when you look at what `EquipmentSlot` is, but specifying `equipped` makes that immediately clear without being that verbose.

Reply:
The arguments make it immediately clear, you don't need additional verbosity to make it clear. `entity.hasStack(EquipmentSlot.CHEST)` == "Has stack in chest slot"

Reply:
It's also worth noting that the "side-effects" only exist for 2 entity types, player and armor stand. For every other living entity, it is merely a standard setter.

Reply:
A better option here may be `hasItem`, as technically there is always a stack in the slot, just that the stack can be empty.

Reply:
I don't think you should rely on arguments that much, especially when a call can look like `hasStack(var7)`.

Reply:
Are we really mapping code based on decompiler generated garbage now?

Reply:
Considering what yarn is used for, I think it is important yes. Even discounting that, I think the code should stay clear when people shorten their `EquipmentSlot` variables to `slot`, although it could be argued that user code should not be taken into account.

Reply:
Even if a variable were simplified to `slot`, `entity.getStack(slot)` reads perfectly fine and you are able to look at the type of `slot` if needed. This would make the method naming consistent with that of an inventory e.g `inventory.getStack(slot)` where `slot` is an `int`.

Reply:
I do think it is an issue to have the same method name for equipment and inventory methods. See `MinecartChestEntity`, which is both an entity and an inventory - it already has a `getStack(int)` method.

Reply:
I feel it is a non-issue when the argument of `EquipmentSlot` very clearly distinguishes it.